src/environmentd/tests/pgwire.rs

```diff
@@ -566,7 +566,7 @@ fn pg_test_inner(dir: PathBuf) {
             _ => panic!("only tcp connections supported"),
         };
         let user = config.get_user().unwrap();
-        let timeout = Duration::from_secs(60);
+        let timeout = Duration::from_secs(if cfg!(coverage) { 120 } else { 60 });
```

Comment:
Eh, maybe just bump this to 120 unconditionally? The fewer conditionals the better IMO. I'm happy for our timeouts to be tuned for our slowest execution environment.